packages/solidstart/src/server/utils.ts

```diff
@@ -0,0 +1,50 @@
+import { flush } from '@sentry/node';
+import { logger } from '@sentry/utils';
+import type { RequestEvent } from 'solid-js/web';
+import { DEBUG_BUILD } from '../common/debug-build';
+
+/**
+ * Takes a request event and extracts traceparent and DSC data
+ * from the `sentry-trace` and `baggage` DSC headers.
+ */
+export function getTracePropagationData(event: RequestEvent | undefined): {
+  sentryTrace: string | undefined;
+  baggage: string | null;
+} {
+  const request = event && event.request;
+  const headers = request && request.headers;
+  const sentryTrace = (headers && headers.get('sentry-trace')) || undefined;
+  const baggage = (headers && headers.get('baggage')) || null;
```

Comment:
L: is there a reason why one is set to `null` and one to `undefined`?

Reply:
I'm not sure tbh, the types were already like this. I think I could change it both to `null` but no strong feelings.